include/circt/Dialect/SystemC/SystemCStatements.td

```diff
@@ -140,6 +140,14 @@ def InstanceDeclOp : SystemCOp<"instance.decl", [
     /// An InstanceDeclOp may not optionally define a symbol.
     bool isOptionalSymbol() { return false; }
 
+    //===------------------------------------------------------------------===//
```

Comment:
I think this method comes from `HWInstanceLike` with this change?

Reply:
(the non-plural version)

Reply:
There is no default declaration for it, but the trait has a default implementation which works if this method is present.